addons/common/fnc_compatibleMagazines.sqf

```diff
@@ -6,20 +6,20 @@ Description:
     Retrieves a list of magazines that are compatible with a weapon.
 
 Parameters:
-    _weapon - Weapon configName or config
-    _allMuzzles - Get magazines for all muzzles on this weapon (default: false)
+    _weapon     - Weapon class name or config <STRING, CONFIG>
+    _allMuzzles - Get magazines for all muzzles on this weapon (optional, default: false) <BOOLEAN>
```

Comment:
Good improvements on the function doc. The type is just called <BOOL> in SQF though.

https://community.bistudio.com/wiki/typeName